app/main/routes.py

```diff
@@ -113,6 +124,58 @@ def add_raid_player():
         flash('RaidPlayer added successfully!')
         return redirect(url_for('main.debug'))
 
+@bp.route('/signups/populate')
+def populate():
```

Comment:
This is awesome!

Now instead of using your "SignUp" Model we want to populate/update the tables Raids, Players, and RaidPlayer. More detail in my comment to this PR!